src/components/prefab-screens.js

```diff
@@ -34,10 +39,23 @@ const defineMessageScreen = (
       this.img = img;
       this.heading = heading;
       this.primaryAction = primaryAction;
-      this.onPrimaryAction = onPrimaryAction;
-      this.secondaryAction = secondarAction;
-      this.onSecondaryAction = onSecondaryAction;
+      this.secondaryAction = secondaryAction;
+      if (closeOnClick) {
```

Comment:
I would like to make this as stupid as it needs to be - WDYT: 
Remove the logic for closeOnClick as it only applys for the actions - and use a decorator for the ones where we need that functionality. 
```js
const closeAfter (f) => { 
          if(f){
            f();
          }
          window.close();
} 
```

```js
const defineMessageScreen = (
   ...
  onSecondaryAction = () => closeAfter (()=>open(sumoLink)),
)
```

Imho that is nicer to read then, as we dont need to recall what that one bool is flagging
```
defineMessageScreen( ... false|true ... )

Reply:
I've updated this. Not my personal style, but feels more appropriate given this project's overall codebase.